service.subtitles.legendasdivx/default.py

```diff
@@ -0,0 +1,5 @@
+class Main:
```

Comment:
Where does the service come from?

Reply:
just went back ... 
bad file.

Reply:
playing Monopoly and coding is not good :)

Reply:
Done this part.
Had to stop the monopoly :)